docs/user/updating.md

```diff
@@ -0,0 +1,25 @@
+# Updating
+
+VSCode will update the Cursorless extension automatically. In the rare case that you have extension auto-updating disabled, you will have to [update the extension manually](https://code.visualstudio.com/docs/editor/extension-marketplace#_update-an-extension-manually).
+
+## Updating the Talon side
+
+The Talon side of Cursorless always needs to be updated manually, simply by performing a `git pull`.
+
+### Linux & Mac
+
+Assuming the default install location of `~/.talon/user`:
+
+```insert code:
+cd ~/.talon/user
```

Comment:
cursorless-talon typically isn't installed in the ~/.talon/user root. Did you mean to include cursorless in this path?

Reply:
Hmm, seems that this is the correct location per `installation.md`?

https://github.com/cursorless-dev/cursorless/blame/main/docs/user/installation.md#L16-L21

Reply:
🤦 my bad, I understand the comment now.